src/probe.bpf.c

```diff
@@ -120,25 +120,25 @@ int trace_egress(struct trace_event_raw_net_dev_xmit* ctx)
   u8* record = buf;
   u32 record_len = __builtin_elementwise_min(SCRATCH_BUF_LEN, data_len);
   bpf_probe_read_kernel(record, record_len, data);
-  s64 len = format_dns_record_to_domain_name(state->name, NAME_BUF_SIZE, record, record_len);
+  s64 len = format_dns_record_to_domain_name(state->domain_name, DOMAIN_NAME_BUF_SIZE, record, record_len);
 
   if (len < 0) {
     bpf_map_delete_elem(&query_state, &key);
     return EXIT_FAILURE;
   }
 
   pid_t pid = bpf_get_current_pid_tgid() >> 32;
-  state->pid = pid;
+  state->tid = pid;
```

Comment:
does tid = tgid?

Reply:
No, this was me changing to using userspace terminology. I think this is going to be _way_ less confusing to users who more likely than not won't be familiar with the kernel terminology.

`tid` (userspace) = `pid` (kernel)
`pid` (userspace) = `tgid`(kernel)